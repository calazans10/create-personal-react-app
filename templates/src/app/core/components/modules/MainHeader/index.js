import React from 'react';
import MainNavigation from '../MainNavigation';
import './index.scss';

const MainHeader = () => (
  <header className="main-header">
    <h1 className="main-header__title">Admin</h1>
    <MainNavigation />
  </header>
);

export default MainHeader;
