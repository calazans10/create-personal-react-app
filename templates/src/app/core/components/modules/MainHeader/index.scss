@import 'styles/utilities';

.main-header {
  display: flex;
  align-items: center;
  padding: rem(16) rem(32);
  background-color: #2f3542;

  &__title {
    color: #fff;
    font-size: rem(28);
    text-transform: uppercase;
  }
}
