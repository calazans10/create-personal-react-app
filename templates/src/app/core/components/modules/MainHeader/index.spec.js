import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MainHeader from './index';

describe('MainHeader', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<MainHeader />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
