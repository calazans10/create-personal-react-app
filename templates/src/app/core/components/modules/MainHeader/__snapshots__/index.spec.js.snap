// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainHeader renders without crashing 1`] = `
<header
  className="main-header"
>
  <h1
    className="main-header__title"
  >
    Admin
  </h1>
  <MainNavigation />
</header>
`;
